src/bindings/python/PyExponentTransform.cpp

```diff
@@ -124,9 +125,10 @@ OCIO_NAMESPACE_ENTER
         {
             OCIO_PYTRY_ENTER()
             ConstExponentTransformRcPtr transform = GetConstExponentTransform(self);
-            std::vector<float> data(4);
-            transform->getValue(&data[0]);
-            return CreatePyListFromFloatVector(data);
+            double data[4];
+            transform->getValue(data);
+			std::vector<double> dataVec{ data, data + sizeof(data) / sizeof(double)  };
```

Comment:
[minor] This line has tab indent but neighbors use spaces.

Reply:
tabs have been replaced by spaces